pfs.ipmu.jp/usage.tex

```diff
@@ -11,125 +11,115 @@
 or science team member. 
 
 \subsubsection{Technical team member}
 
-You need to get pfs.ipmu.jp (https) account at least, 
-and you can request account for JIRA and gitolite if you will work on software 
-development. 
-Ask local manager at your institution 
-to send following information to \url{pfsprojectoffice@ipmu.jp}. 
+You need at least to request an account to log in to \url{https://pfs.ipmu.jp}.
+In addition, if you work for software development, you can optionally 
+request a member privilege of GitHub: this is needed for you to access 
+to private repositories. 
+GitHub is also used for an issue tracking system for the development of 
+ICS (Instrument Control Software). Software documentation (ics\_doc) 
+repository is also on GitHub as a private repository. 
+
+To make these requests, please ask the local site manager at your institute 
+to send the following information to \url{pfsprojectoffice@ipmu.jp}. 
 
 \begin{itemize}
   \item Your contact email address.
-  \item Preferred account name.
+  \item Preferred account name for pfs.ipmu.jp.
   \item Photo for photo directory at pfs.ipmu.jp. (JPEG, less than 50kB; width 
     250--300pixel, height 250--350pixel)
-  \item (If you will work on software development) Ask enabling JIRA option.
-  \item (If you need to push/pull git repositories) Your ssh public key.
+  \item (If you are software developers or related workers) Your GitHub account name
 \end{itemize}
 
-If you already have pfs.ipmu.jp account but have not enabled JIRA option, 
-email to \url{jira@pfs.ipmu.jp}. 
+For the Git repositories operated at GitHub, 
+there is one (mostly private) team per institute where the members have 
+the admin privilege. 
+You can ask a member of the team in your institute to add yourself and 
+get the admin privilege to the repositories related to the development 
+activities at your institute. Adding you to the team is as 'invitation', 
+and you will need to approve an invitation via email. 
+
+Meanwhile, JIRA is used as a management tool for the development of 
+the other PFS software packages than ICS, such as DRP (Data Reduction 
+Pipeline), ETS (Exposure Targeting Software) and SPT (Survey Planning 
+and Tracking software). PFS JIRA (https://pfspipe.ipmu.jp) is open to 
+the public, but is operated in an integration of single sign on (SSO) with 
+other services (LDAP service of pfs.ipmu.jp), you need to use the same 
+account name and password. If you already have an account at pfs.ipmu.jp 
+and have not logged in to PFS JIRA before, you need to contact to 
+\url{jira@pfs.ipmu.jp} to activate your account in PFS JIRA.
 
-If you already have pfs.ipmu.jp account but have not registered ssh public 
-key to git repository service, send your ssh public key to 
-\url{git-admin@pfs.ipmu.jp} (refer sec.~\ref{sec:pfs-gitolite}).
 
 \subsubsection{Science team member}
 
 If you don't have SuMIRe/PFS pbworks wiki account, first request an account 
```

Comment:
It seems that you are mixing the procedure for requesting account and description of service itself in these sections.
If one is a science team member, he or she will skip the section for technical one, and will not find how to request the account.
It would be better to move the procedure of requesting an account for pfs.ipmu.jp to section "registration".